pkg/minikube/cluster/cluster.go

```diff
@@ -129,6 +129,7 @@ func DeleteHost(api libmachine.API) error {
 	m := util.MultiError{}
 	m.Collect(host.Driver.Remove())
 	m.Collect(api.Remove(constants.MachineName))
+
```

Comment:
Please remove this extra white line.

Reply:
Agreed. Looks like unnecessarily got added.

Reply:
Ops.... My bad...